packages/react-auth/package.json

```diff
@@ -28,7 +28,8 @@
   "license": "MIT",
   "packageManager": "yarn@1.22.22",
   "scripts": {
-    "build": "parcel build && cp dist/index.d.ts dist/index.d.mts",
+    "build": "yarn build-docs && parcel build && cp dist/index.d.ts dist/index.d.mts",
+    "build-docs": "yarn typedoc --plugin typedoc-plugin-rename-defaults",
```

Comment:
Specifying the typedoc-plugin-rename-defaults plugin can be moved to `typedoc.json`.